code/game/turfs/simulated.dm

```diff
@@ -83,7 +83,7 @@
 
 	if (istype(A,/mob/living))
 		var/mob/living/M = A
-		if(M.lying)
+		if(M.lying || M.flying)

```

Comment:
//VOREStation Edit

Reply:
Done, that's what I get for doing that at 3 am.